cfgov/jinja2/v1/_includes/organisms/sidebar-breakout.html

```diff
@@ -0,0 +1,118 @@
+{# ==========================================================================
+
+   sidebar_breakout.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates Sidebar Breakout markup when given:
+
+   settings:               An object used to customize the markup.
+
+   settings.heading:       A string containing heading text.
+
+   settings.image:         An object used to customize the image
+                           markup.
+
+   settings.intro.body:    A string containing body text.
+
+   settings.intro.heading: A string containing heading text.
+
+   settings.related_posts: An object used to customize the related posts
+                           markup.
+
+   settings.text_intro:    The text intro markup.
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/f81f8ad946f646c827668b7520c765b54dbf2ddb